Migration docs revamp. (#24630)
dotnet/docs: docs/core/porting/premigration-needed-changes.md

---
title: Needed changes before porting code
description: Make the necessary changes to build targeting .NET Core.
author: stebon
ms.date: 06/10/2021
---
# Overview
[content needed]

> # Upgrade to required tooling

Reviewer: Needs to be ##

```suggestion
## Upgrade to required tooling
```

---

> title: Needed changes before porting code
> description: Make the necessary changes to build targeting .NET Core.

Reviewer: Again, these should be pretty descriptive. Also, I wonder if a more action-focused title would be good? Maybe it sounds too blog-like?
```suggestion
title: Do these things before porting code
description: Upgrade your dev environment to the latest version, and upgrade your project files. You must do these things before porting your code from .NET Framework.
```

author: stebon
ms.date: 06/10/2021
---

> # Overview

Reviewer: Need a clearer title here. This should be unique across all articles
```suggestion
# Premigration overview
```
